### DISCUSSION TITLE
Smart Meter Interface / eBus example

### DISCUSSION THREAD

**Discussion by ceraz68** [arendst/Tasmota#13713] [Q&A] [answered]
Hello does anyone have a reference I can read up on making use of Smart Meter to directly connect to an ebus interface from a Sonoff or Wemos?

I'm looking to interface to a brink ventilation system W400 (like renovent, wolf etc.)

I read the vBus example but the two pin eBus works on a different voltage [(bus wikipedia)](https://en.wikipedia.org/wiki/EBUS_(serial_buses))

Willing to experiment if you have any tips,

Thanks a lot :-)

**Comment by barbudor** [comment, marked as answer]
As far as you can get a converter from ebus to standard 3.3v level that's a standard asynchronous serial signal

**Reply by barbudor** [reply]
This project should be usable with tasmota
https://ebus.github.io/adapter/index.en.html

**Reply by ceraz68** [reply]
Thanks for that, as far I as understand this ebus adapter is to be combined with the ebusd code that is Mqtt enabled. I was going to create an Mqtt heater accessory using homebridge, so in may case it probably means that I won't need to use Tasmota smart meter.

I see they have a version 3:  https://adapter.ebusd.eu/index.en.html

**Reply by barbudor** [reply]
The adpater v2 is meant to be directly used wiht a Wemos D1-mini ESP8266 board. So you could use Tasmota and it's SML feature

The v3 is more complex and can be used with different options such as Wemod D1, but also include USB serial adapter to be connected to a PC or Raspberry Pi and other options.

I think the V2 should be enough, but I haven't looked into details.
